src/PowerShellEditorServices/Language/CommandHelpers.cs

```diff
@@ -14,6 +16,20 @@ namespace Microsoft.PowerShell.EditorServices
     /// </summary>
     public class CommandHelpers
     {
+        private static HashSet<string> NounBlackList =
+            new HashSet<string>
+            {
+                "Module",
```

Comment:
Does this mean we won't get info for Import-Module?  That is a pretty commonly used command.  The rest, less so.  But if it is temporary, then let's try it and see if folks complain about missing info for these nouns.  

I wonder if it would be worth returning some static text for these nouns like "Intellisense for this noun disabled due to perf issue, will be fixed in the future." or something like that?